### PR TITLE
Clarify that tintColor sets the ProgressBar's filled part

### PR DESCRIPTION
**Pull request by FokkeZB**
Also fix mixed usage of text and message.

### CODE REVIEW

**Comment by FokkeZB** [comment]
@clinseman

File: apidoc/Titanium/UI/ProgressBar.yml [hunk 1]
```diff
@@ -71,13 +71,13 @@ properties:
   - name: message
     summary: Progress bar text.
     type: String
-
+  - name: tintColor
+    summary: The color shown for the portion of the progress bar that is filled.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/ProgressBar.yml]: Missing `type: String` and `platforms: [iphone,ipad]` keys.

Comment (FokkeZB) [inline apidoc/Titanium/UI/ProgressBar.yml]: Doesn't it inherit that from the parent? (Ti.UI.View)

Comment (hansemannn) [inline apidoc/Titanium/UI/ProgressBar.yml]: When you redefine the property, it will (as far as I know) override the "parent" key.

Comment (FokkeZB) [inline apidoc/Titanium/UI/ProgressBar.yml]: Are you sure? This View also overrides the add-method and here it doesn't redefine properties it doesn't change either:
https://github.com/FokkeZB/titanium_mobile/blob/patch-41/apidoc/Titanium/UI/ProgressBar.yml#L40

**Comment by hansemannn** [comment]
You are correct, just checked it by generating the docs. Approved!

**Pull request merged**